lib/brook/storage/redis/migration.ex

```diff
@@ -0,0 +1,53 @@
+defmodule Brook.Storage.Redis.Migration do
+  alias Brook.Storage.Redis
+
+  @expiration 60 * 60 * 24 * 7
+
+  def migrate(instance, redix, namespace) do
+    regex = ~r/^#{namespace}:(?<collection>[[:alnum:]-_]+):(?<key>[[:alnum:]-_]+)$/
+
+    view_state_entries =
+      keys(redix, "#{namespace}:*")
+      |> Enum.filter(&Regex.match?(regex, &1))
+      |> Enum.map(fn key -> Regex.named_captures(regex, key) |> Map.put("redis_key", key) end)
+      |> Enum.map(&get_old_view_state(redix, &1))
+      |> Enum.map(&get_old_events(redix, &1))
+
+    Enum.each(view_state_entries, &save_view_state(instance, &1))
+    Enum.each(view_state_entries, &move_old_entry(redix, &1))
+  end
+
+  defp keys(redix, pattern), do: Redix.command!(redix, ["KEYS", pattern])
```

Comment:
This is a nitpick and feels weird saying, but is it worth pulling this guy out to it's own private function. I think it may read better just inline above.